src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -39,11 +40,17 @@ export class AssetsTable extends React.Component {
         key: 'date_added',
         columnSortable: true,
       },
+      urls: {
+        label: 'URLs',
+        key: 'urls',
+        columnSortable: false,
+      },
```

Comment:
Note to @efischer19 - this is going to break the test added in https://github.com/edx/studio-frontend/pull/48 because you used brittle hard-coded expected values. Figure out a smarter way to write that test.